### DISCUSSION TITLE
Problems running singlepoint opt

### DISCUSSION THREAD

**Discussion by S5331107** [mdolab/MACH-Aero#118] [General]
I´m trying to run the tutorial of single point optimization for NACA 0012 airfoil and i´m not able to make it work. Error appear, related to dimension of arrays. In attachment screenshot of the message.
I´m running the simulation from scratch. 
![Screenshot from 2023-10-23 16-50-14](https://github.com/mdolab/MACH-Aero/assets/121494566/266d0e5e-4a7d-4c07-b256-0c65d6d75466)

**Comment by marcomangano** [comment]
Hello, this looks related to this issue: https://github.com/mdolab/MACH-Aero/issues/96. 
The recommended solution mentioned [here](https://github.com/mdolab/MACH-Aero/issues/96#issuecomment-1582407845) is to roll back to pyoptsparse v 2.10.0

**Comment by S5331107** [comment]
@marcomangano Thank you so much, problem solved!
